shared/bundle_analysis/report.py

```diff
@@ -0,0 +1,122 @@
+import os
+import re
+import sqlite3
+import tempfile
+from typing import Any, Dict, Iterator, Optional
+
+from sqlalchemy.exc import OperationalError
+from sqlalchemy.orm import Session
+from sqlalchemy.sql import func
+
+from . import models
+from .parser import parse
+
+# matches file paths with content hashes like:
+# path/to/file-aa21a697.ext
+CONTENT_HASH_RE = re.compile(r"^.+(-[\w\d]{8})\.[\w\d]+$")
```

Comment:
Abhi pointed me to https://github.com/getsentry/relay/blob/24b39d09d7f59f95c3cbb37fed0839ce29e33ec3/relay-event-normalization/src/regexes.rs#L66 which looks a bit more comprehensive in terms of the hash part of the regex

Reply:
This sort of normalization is now happening in the JS plugin that generates the stats JSON